src/Ocelot.Provider.Polly/PollyQoSResiliencePipelineProvider.cs

```diff
@@ -76,8 +82,8 @@ protected virtual ResiliencePipelineBuilder<HttpResponseMessage> ConfigureCircui
         var info = $"Circuit Breaker for the route: {GetRouteName(route)}: ";
         var strategyOptions = new CircuitBreakerStrategyOptions<HttpResponseMessage>
         {
-            FailureRatio = 0.8,
-            SamplingDuration = TimeSpan.FromSeconds(10),
+            FailureRatio = options.FailureRatio,
+            SamplingDuration = TimeSpan.FromMilliseconds(options.SamplingDuration),
```

Comment:
Both `SamplingDuration` and `BreakDuration` are `TimeSpan`s, so they can be initialized from milliseconds.
Not an issue! FYI